product_customer_code_picking/README.rst

```diff
@@ -0,0 +1,42 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=======================================
+Product Customer Code For Stock Picking
+=======================================
+Based on product_supplierinfo_for_customer, this module loads in every Stock Picking the
```

Comment:
To be complete, 'in every Stock Picking **at the move level**...' or something like that to help people to find the info